rustls/src/conn.rs

```diff
@@ -587,7 +598,12 @@ impl ConnectionCommon {
     }
 
     pub fn reader(&mut self) -> Reader {
-        Reader { common: self }
+        Reader {
+            received_plaintext: &mut self.common_state.received_plaintext,
+            /// Are we done? i.e., have we processed all received messages, and received a
+            /// close_notify to indicate that no new messages will arrive?
+            connection_at_eof: !self.peer_eof || self.message_deframer.has_pending(),
```

Comment:
The name of this value doesn't seem to match it's behavior (should it be connection_not_at_eof?)

Reply:
This was fixed, thanks for the careful review!